packages/flutter/lib/src/animation/animation_controller.dart

```diff
@@ -689,11 +681,12 @@ class AnimationController extends Animation<double>
   /// The most recently returned [TickerFuture], if any, is marked as having been
   /// canceled, meaning the future never completes and its [TickerFuture.orCancel]
   /// derivative future completes with a [TickerCanceled] error.
-  TickerFuture fling({ double velocity = 1.0, SpringDescription? springDescription, AnimationBehavior? animationBehavior }) {
+  TickerFuture fling(
+      {double velocity = 1.0, SpringDescription? springDescription, AnimationBehavior? animationBehavior}) {
     springDescription ??= _kFlingSpringDescription;
     _direction = velocity < 0.0 ? _AnimationDirection.reverse : _AnimationDirection.forward;
-    final double target = velocity < 0.0 ? lowerBound - _kFlingTolerance.distance
-                                         : upperBound + _kFlingTolerance.distance;
+    final double target =
+        velocity < 0.0 ? lowerBound - _kFlingTolerance.distance : upperBound + _kFlingTolerance.distance;
```

Comment:
These changes are concerning.

Maybe adding a trailing comma on `animationBehavior` would make it a little better, but the ternary would be more readable if it was three lines.

Reply:
Better with a trailing comment:

```dart
  final double target = velocity < 0.0 //
      ? lowerBound - _kFlingTolerance.distance
      : upperBound + _kFlingTolerance.distance;
```